doc/news/_preparation_next_release.md

```diff
@@ -44,18 +44,8 @@ docker run -it elektra/elektra
 
 ## Highlights
 
-- <<HIGHLIGHT>>
-- <<HIGHLIGHT>>
-- New Changetracking API
-- ODBC Backend _(Florian Lindner @flo91)_
-- <<HIGHLIGHT>>
-
-### <<HIGHLIGHT>>
-
 - Session Recording
-- New Changetracking API
-- New spec plugin
-- <<HIGHLIGHT>>
+- ODBC Backend _(Florian Lindner @flo91)_
 
 ### Session Recording
```

Comment:
Can you maybe do a small https://asciinema.org with how recording works? (we already did in earlier releases)